src/js/view/controls/components/chapters.mixin.js

```diff
@@ -35,15 +39,29 @@ const ChaptersMixin = {
         if (Array.isArray(data)) {
             data.forEach((obj) => this.addCue(obj));
             this.drawCues();
+            const existingCues = this._model.get('cues');
+            const playerCues = existingCues.concat(data);
+            this._model.set('cues', playerCues);
```

Comment:
Why do you draw the cues and then update the model?

Reply:
I'm not sure why it would matter. Can you explain?

Reply:
Refactored this to just set the cue to the model when the chapters are loaded. Changing the cues on the model will then call addCue() and drawCues().